Google.PowerShell/ReleaseFiles/GoogleCloudPowerShell.psd1

```diff
@@ -64,22 +64,22 @@ ProcessorArchitecture = 'None'
 # TypesToProcess = @()
 
 # Format files (.ps1xml) to be loaded when importing this module
-FormatsToProcess = @("..\PowerShell\GoogleCloud\GoogleCloudPlatform.Format.ps1xml")
+FormatsToProcess = '..\PowerShell\GoogleCloud\GoogleCloudPlatform.Format.ps1xml'
 
 # Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
-NestedModules = @('..\PowerShell\GoogleCloud\Google.PowerShell.dll')
+NestedModules = @()
 
 # Functions to export from this module
-FunctionsToExport = '*'
+# FunctionsToExport = @()
```

Comment:
There are automatically generated from New-ModuleManifest cmdlet and when we don't supply anything, they will be generated like this.